tests/integration/components/ui-table/cell/cell-local-date-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | ui-table/cell/cell-local-date', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`{{ui-table/cell/cell-local-date}}`);
+
+    assert.equal(this.element.textContent.trim(), '');
+
+    // Template block usage:
+    await render(hbs`
+      {{#ui-table/cell/cell-local-date}}
+        template block text
+      {{/ui-table/cell/cell-local-date}}
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
+  });
+});
```

Comment:
Change the component test file. This is just the default file you get when you generate a component.

Reply:
Okay. Doing so